README.md

```diff
@@ -10,7 +14,7 @@ Naming and functionality are still subject to *breaking* changes.
 
 ***
 
-This document shows how to deploy and use the CyberArk Sidecar Injector [MutatingAdmissionWebhook](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19) Server which injects sidecar container(s) into a pod prior to persistence of the underlying object.
+This document shows how to deploy and use the CyberArk Sidecar Injector [mutating admission webhook controller](https://kubernetes.io/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19) which injects sidecar container(s) into a pod prior to persistence of the underlying object.
```

Comment:
Is there a better link than a link to the beta page?

Reply:
yup updated